npm/private/npm_translate_lock.bzl

```diff
@@ -432,11 +432,14 @@ def _impl(rctx):
         else:
             fail("rules_js internal validation error, please file an issue")
 
-        rctx.execute([
+        result = rctx.execute([
             rctx.path(Label("@nodejs_host//:bin/node")),
             rctx.path(Label("@pnpm//:package/bin/pnpm.cjs")),
             "import",
         ])
+        if result.return_code:
+            msg = "pnpm import failed: \nSTDOUT:\n%s\nSTDERR:\n%s" % (result.stdout, result.stderr)
```

Comment:
worth putting the error code in?

Reply:
sure, maybe we should do that everywhere else too

Reply:
I assume the error code of external tools might be meaningful sometimes 🤷